### DISCUSSION TITLE
Feature Request: Item Banning/Whitelisting

### DISCUSSION THREAD

**Discussion by TheBlueKingLP** [Zrips/CMI#1827] [Feature Requests]
**Description of issue or feature request:**
I would like to have a feature similar to essentials anti build.
The features:
Ban item per world.
Ban item per action(only ban placing it, only ban breaking it, or only ban using it
Whitelist item per world
Whitelist item per action(same as above but whitelisting it)

---

**Comment by 9dolanla** [comment]
I wish cmi could have this feature too
